src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  - /docs/using-new-relic/welcome-new-relic/getting-started/introduction-new-relic
+  - /taxonomy/term/5766
+  - /docs/using-new-relic/welcome-new-relic/getting-started/what-new-relic-can-do-you
+  - /docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-one-transition-guide-july-2020
+  - /docs/using-new-relic/welcome-new-relic/get-started/aW50cm9kdW
+  - /docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic
+  - /docs/using-new-relic/
+  - /docs/using-new-relic/welcome-new-relic
+  - /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
+  - /docs.newrelic.com/docs/new-relic-solutions/new-relic-one/ui-data/new-relic-feature-end-life-announcements-july-2020
+  - /docs/new-relic-solutions/new-relic-one/install-configure/install-new-relic
+  - /docs/new-relic-solutions/get-started/quick-launch-guide
+  - /docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability
+  - /docs/agents/manage-apm-agents/installation/install-agent
+  - /docs/new-relic-one/use-new-relic-one/cross-product-functions/install-configure/install-new-relic
+  - /docs
+  - /docs/using-new-relic/cross-product-functions/install-configure/install-new-relic
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: 2024-03-dd
+---
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+import NRDashboardsMain from 'images/dashboards_screenshot-crop_main.webp'
+
+import NRErrosInboxMain from 'images/errors-inbox_screenshot-crop_main.webp'
+
+import NRIastMain from 'images/iast_screenshot-crop_main.webp'
+
+import NRInfraMain from 'images/infrastructure_screenshot-crop_main.webp'
+
+import NRLambdaMain from 'images/lambda_screenshot-crop_main.webp'
+
+import NRLiveArchivesMain from 'images/live-archives_screenshot-crop_main.webp'
+
+import NRLogsMain from 'images/logs_screenshot-crop_main.webp'
+
+import NRK8sMain from 'images/kubernetes_screenshot-crop_main.webp'
+
+import NRMobileMain from 'images/mobile_screenshot-crop_main.webp'
+
+import NRNetworkMain from 'images/network_screenshot-crop_main.webp'
+
+import NROpenTelemetryMain from 'images/opentelemetry_screenshot-crop_main.webp'
+
+import NRServiceLevelsMain from 'images/slm_screenshot-crop_main.webp'
+
+import NRSyntheticMain from 'images/synthetic_screenshot-crop_main.webp'
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
+Welcome to the New Relic observability platform! Our platform gives you deep visibility into your systems. It helps you solve problems you may not have been aware of, regardless of the technology you use. You can see how New Relic works [here](#how-it-works) or [get started](#get-started) now if you want to jump right in.
```

Comment:
The intro is friendly, but I think instead of linking users to a section that's halfway down the doc, we can spend a little time giving a high level overview of what New relic can do. Here's something I whipped up really quick: 

> New Relic is an observability platform that gives you direct access to your system’s behavior. Whether you work primarily in distributed systems, the cloud, or traditional server-based architectures, New Relic can collect data that helps you solve problems before they occur. 
> 
> How you get started depends on your system, but you’ll follow the same general pattern: you’ll instrument a technology through installation, then use the platform to set up alerts, dashboards, and other capabilities. 
> 
> Before we dive in, make sure you’ve [signed up for an account](SIGN UP LINK). It’s free, forever.

Reply:
altho I like the impulse to have a welcome statement before we "get down to business" too.

Reply:
Thank you, @akristen for your comment. I had that approach several commits ago ;-). But Austin and Shawn prefer this way.